### ISSUE TITLE
[firebase_messaging] IOS stopped receiving push notifications

### ISSUE THREAD

**Issue by tcerven** [FirebaseExtended/flutterfire#1408]
Push notifications in IOS just stopped working in an app that had been working fine for months. Things that have changed are 1) upgraded flutter 2) upgraded Mac OS to 10.15. This occurs with both iOS 12 and 13. **Android still works.**

Flutter doctor:
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.15.1 19B88, locale en-US)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.2)
[✓] Android Studio (version 3.5)
[!] IntelliJ IDEA Community Edition (version 2019.2.3)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.34.0)
[✓] Connected device (4 available)

! Doctor found issues in 1 category.
```

**Comment by tcerven** [comment]
I created a new app using the example in firebase_messaging and it worked! I compared this with my non-functioning app and the only difference I could find is that the one that works had this in the pubspec:

```flutter:
  plugin:
    androidPackage: io.flutter.plugins.firebasemessaging
    iosPrefix: FLT
    pluginClass: FirebaseMessagingPlugin
```
and the one that doesn't work do not have it in the pubspec. I tried putting it in there but got a podfile error:
```
[!] No podspec found for `the_ions_app` in `.symlinks/plugins/the_ions_app/ios`
```

**Comment by nektar95** [comment]
same problem, also after adding

>   plugin:
    androidPackage: io.flutter.plugins.firebasemessaging
    iosPrefix: FLT
    pluginClass: FirebaseMessagingPlugin

i got error 
`Couldn't read file LocalFile: .../android/src/main/kotlin/io/flutter/plugins/firebasemessaging/FirebaseMessagingPlugin.kt' even though it exists. Please verify that this file has read permission and try again.`

**Comment by tcerven** [comment]
I worked around the problem by brute force. Since the example app works, I used that as a start. Then I removed the plugin section of the pubspec:
```
plugin:
androidPackage: io.flutter.plugins.firebasemessaging
iosPrefix: FLT
pluginClass: FirebaseMessagingPlugin
```
and replaced it with the package:
```
firebase_messaging: ^5.1.8
```
I also included all my pubspec packages and retested just to make sure it was still working. Then I copied in the correct google services files SDK files and changed the application/bundle IDs to match.  Then I copied and pasted in my source and the other source and asset files and finally, APN working again.

I don't know what caused the problem but a lot of people are having it. Until a better solution is found, don't waste your time if you don't have it looking for one, just do the work around. I spent many hours searching and trying different things and got nowhere.
